### DISCUSSION TITLE
citar-open-notes to create new note according to template?

### DISCUSSION THREAD

**Discussion by maasdorp** [emacs-citar/citar#411] [Q&A] [answered]
If everything is set correctly, should citar-open-notes create a new note if no associated note exists in the citar-notes-paths directory? 
I can open existing notes, but cannot create new notes according to the default template.

**Comment by bdarcus** [comment, marked as answer]
Yes, but there was a regression. See #412.

**Reply by maasdorp** [reply]
Thanks, will keep my eye on #412
